std_msgs/Header header

float32 time
string host
int32 count
float32 timeout
# % of packets loss
float32 packet_loss
# min transmission time
float32 t_min
# average transmission time
float32 t_avg
# max transmission time
float32 t_max
float32 t_mdev
